rotkehlchen/chain/evm/node_inquirer.py

```diff
@@ -1379,9 +1379,14 @@ def get_blocknumber_by_time(
             closest: Literal['before', 'after'] = 'before',
     ) -> int:
         """Searches for the blocknumber of a specific timestamp
-
+        - Performs the blockscout api call by default first
+        - If RemoteError gets raised it queries etherscan
         May raise RemoteError
         """
+        if self.blockscout:
+            with suppress(RemoteError):
+                return self.blockscout.get_blocknumber_by_time(ts, closest)
```

Comment:
With the suppression here, do we log anywhere that blockscout fails and now we are falling back to etherscan.

Reply:
yes, it gets logged ad the blockscout query logic